src/components/comparison/layers-panel/list-item/list-item.tsx

```diff
@@ -0,0 +1,223 @@
+import styled, { css } from "styled-components";
+
+import { Checkbox } from "../../../checkbox/checkbox";
+import { RadioButton } from "../../../radio-button/radio-button";
+
+import { color_brand_quaternary } from "../../../../constants/colors";
+import { ExpandIcon } from "../../../expand-icon/expand-icon";
+
+type ListItemProps = {
+  id: string;
+  parentId?: string;
+  title: string;
+  icon?: React.ReactNode;
+  listItemType: ListItemType;
+  optionsContent?: JSX.Element;
+  expandState?: ExpandState;
+  selected: boolean;
+  isOptionsPanelOpen?: boolean;
+  usePopoverForOptions?: boolean;
+  onClick: (id: string, parentId?: string) => void;
+  onOptionsClick?: (id: string) => void;
+  onExpandClick?: () => void;
+  onClickOutside?: () => void;
+  children?: React.ReactNode;
+};
+
+type ContainerProps = {
+  isChild: boolean;
+  checked: boolean;
+  hasChildren: boolean;
+};
+
+const Container = styled.div<ContainerProps>`
+  display: flex;
+  width: ${({ isChild }) =>
+    isChild ? "calc(100% - 6px)" : "calc(100% - 30px)"};
+  justify-content: space-between;
+  flex-direction: column;
+  align-items: center;
+  padding: 10px 20px 10px 10px;
+  background: transparent;
+  cursor: pointer;
+  margin-bottom: 8px;
+
+  ${({ checked, hasChildren }) =>
+    checked &&
+    !hasChildren &&
+    css`
+      background: ${({ theme }) => theme.colors.mainHiglightColor};
+      box-shadow: 0px 17px 80px rgba(0, 0, 0, 0.1);
+      border-radius: 8px;
+    `}
+
+  ${({ hasChildren }) =>
+    !hasChildren &&
+    css`
+      &:hover {
+        background: ${({ theme }) => theme.colors.mainDimColor};
+        box-shadow: 0px 17px 80px rgba(0, 0, 0, 0.1);
+        border-radius: 8px;
+      }
+    `}
+`;
+
+const ItemContentWrapper = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  width: 100%;
+`;
+
+const OptionsButton = styled.div`
+  width: 16px;
+  height: 16px;
+  display: flex;
+  justify-content: center;
+`;
+
+const OptionsIcon = styled.div`
+  position: relative;
+  width: 4px;
+  height: 4px;
+  background-color: ${color_brand_quaternary};
+  border-radius: 50%;
+  margin-bottom: 12px;
+  &:before,
+  &:after {
+    content: "";
+    position: absolute;
+    width: 4px;
+    height: 4px;
+    left: 0px;
+    background-color: ${color_brand_quaternary};
+    border-radius: inherit;
+  }
+  &:before {
+    top: 6px;
+  }
+  &:after {
+    top: 12px;
+  }
+`;
+
+const ContentWithOptionsWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-direction: row;
+  width: 100%;
+`;
+
+const Title = styled.div`
+  margin-left: 16px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 19px;
+  overflow: hidden;
+  color: ${({ theme }) => theme.colors.fontColor};
+`;
+
+const ControlsWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`;
+
+export const ListItem = ({
+  id,
+  parentId,
```

Comment:
I believe we can get rid of parentId in this component

Reply:
It is easier to provide a prop to components instead of traversing all examples to find a parent example for child one.
Also some styling logic based on that.

Reply:
Removed